packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -119,6 +123,11 @@ export const getSelector = <Component extends (props: any) => JSX.Element | unde
 };
 
 const findSelector = (component: (props: any) => JSX.Element | undefined): any => {
+  /* @conditional-compile-remove(dialpad) */
+  if (component === _Dialpad) {
```

Comment:
Can this be moved into the switch statement below? (or is conditional compilation blocking that?)

Reply:
Yeah this is for conditional compilation